hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -682,6 +682,9 @@ public final class OzoneConfigKeys {
   public static final String
       OZONE_OM_NETWORK_TOPOLOGY_REFRESH_DURATION_DEFAULT = "1h";
 
+  public static final String
+      OZONE_COMPLIANCE_ALLOWED_HDDS_X509_SIGNATURE_ALGO = "ozone.compliance.allowed.hdds.x509.signature.algorithm";
+  public static final String OZONE_COMPLIANCE_ALLOWED_HDDS_X509_SIGNATURE_ALGO_DEFAULT = "*";
```

Comment:
`HDDS_X509_SIGNATURE_ALGO` is defined in the `HddsConfigKeys` class, shouldn't we have this there too, right after that constant?

Reply:
Thank you Zita, you are right I've moved them. Initially I wanted all these config options to land at the end of OzoneConfigKeys to track them but the prefix is a better way to do it.